sdks/python/apache_beam/runners/worker/sdk_worker.py

```diff
@@ -66,7 +66,10 @@ def get_responses():
     for work_request in control_stub.Control(get_responses()):
       logging.info('Got work %s', work_request.instruction_id)
       request_type = work_request.WhichOneof('request')
-      if request_type == ['process_bundle_progress']:
+      # WhichOneOf returns the name of the set field as a single string
```

Comment:
I think the intent was to assign other items to this pool, e.g. liquid sharding request. Maybe change to 

if work_request.WhichOneof('request') in ['process_bundle_request']:
    ...

Reply:
Makes sense